### DISCUSSION TITLE
Cannot install on Mac (unidentified developer)

### DISCUSSION THREAD

**Discussion by Sergeant86** [stashapp/stash#3713] [Q&A] [answered]
Hi,

I'm trying to install Stash via "Terminal", but when I press enter, the operating system won't let me install because the file comes from an **"unidentified developer"**.
When I go to System Preferences, I don't have the option to allow the installation of these kind of files.

Is there a dmg file that makes the process easier?

Thank you in advance.

**Comment by DogmaDragon** [comment, marked as answer]
https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/13.0/mac/13.0
https://docs.stashapp.cc/getting-started/installation/macos/
